src/sentry/api/endpoints/organization_details.py

```diff
@@ -35,6 +35,7 @@
     ScheduledDeletion,
     UserEmail,
 )
+from sentry.scmode import ServerComponentMode
```

Comment:
I'm not sure the `sc` abbreviation is ideal. I would rather we use the `servercomponentmode` or come up with something shorter. Some ideas that came to me were:

* sentry.api.modes
* sentry.servermode
* sentry.api.silos

There are probably several other good names we could come up with.

Reply:
`servermode` seems to strike a nice balance.